README.rst

```diff
@@ -9,30 +9,195 @@ modelmeta
    :target: https://codeclimate.com/github/pacificclimate/modelmeta
    :alt: Code Climate			  
 
-The `modelmeta` package is a Python package that provides an `Object Relational Mapping (ORM) <http://en.wikipedia.org/wiki/Object-relational_mapping>`_ layer for accessing the `Pacific Climate Impacts Consortium (PCIC) <http://www.pacificclimate.org/>`_ (PCIC)'s database of `coverage data <http://en.wikipedia.org/wiki/Coverage_data>`_ metadata. The pacakge provides model classes for each of the tables in the database in order to provide a ... querying 
+Overview
+========
 
-With this package, one can recreate the database schema in `PostgreSQL <http://www.postgresql.org>`_ or `SQLite <http://www.sqlite.org>`_ and/or use the package as an object mapper for programmatic database access. `modelmeta` uses `SQLAlchemy <http://www.sqlalchemy.org>`_ to provide the ORM layer.
+``modelmeta`` is a Python package that provides an
+`Object Relational Mapping (ORM) <http://en.wikipedia.org/wiki/Object-relational_mapping>`_ layer 
+for accessing the `Pacific Climate Impacts Consortium (PCIC) <http://www.pacificclimate.org/>`_'s
+database of `coverage data <http://en.wikipedia.org/wiki/Coverage_data>`_ metadata. 
+The package provides model classes for each of the tables in the database.
 
-The intent of the database itself is to separate the small, inexpensive, structured metadata and attribute information (stored in the database) from the expensive-to-access bulk spatiotemporal data (stored on disk in multidimensional files).
+With this package, one can recreate the database schema in `PostgreSQL <http://www.postgresql.org>`_ 
+or `SQLite <http://www.sqlite.org>`_ and/or use the package as an object mapper for programmatic database access.
 
---------------
-How to Install
---------------
+``modelmeta`` uses `SQLAlchemy <http://www.sqlalchemy.org>`_ to provide the ORM layer, and
+`Alembic <http://alembic.zzzcomputing.com/en/latest/>`_ to manage database creation and migration (see section
+below).
 
-One can install `modelmeta` using the standard methods of any other Python package.
+The intent of the database itself is to separate the small, inexpensive, structured metadata and attribute information 
+(stored in the database) from the expensive-to-access bulk spatiotemporal data (stored on disk in multidimensional 
+files). It provides an efficiently searchable index of the bulk data files, and separates storage from indexing.
 
-1. clone our repository and run the setup script
+Installation
+============
+
+Install ``modelmeta`` using the standard methods for any other Python package.
+
+1. Clone the repository and run the setup script::
 
     $ git clone https://github.com/pacificclimate/modelmeta
     $ cd modelmeta
     $ python setup.py install
```

Comment:
May want to add the pip instructions (e.g. ``pip install -e .`` or ``pip install .``), while we're working on this section.